hw/ip/hmac/doc/registers.md

```diff
@@ -183,11 +182,9 @@ the updated value is discarded.
 ### CFG . key_length
 Key length configuration.
 
-This is a 5-bit one-hot encoded field to configure the key length for HMAC. The HMAC
-supports key lengths of 128-bit, 256-bit, 384-bit, 512-bit and 1024-bit, but will
-to block size if key length is greater than block size, i.e. max of 1024 bits
-for digest size SHA-2 384/512 and respectively 512 bits for SHA-2 256. The value of
-this register is irrelevant when only SHA-2 (not keyed HMAC) is configured.
+This is a 5-bit one-hot encoded field to configure the key length for HMAC.
+The HMAC supports key lengths of 128-bit, 256-bit, 384-bit, 512-bit and 1024-bit, but will to block size if key length is greater than block size, i.e. max of 1024 bits for digest size SHA-2 384/512 and respectively 512 bits for SHA-2 256.
```

Comment:
I don't know what "will to block size" is trying to say here?  (It's carried over from the original documentation, this change was just one-sentence-per-line-ing things but we probably need to sort it out.)

Reply:
thanks, this bit has been updated altogether in a later PR, since we will not default to block size anymore. If configuration is invalid, we signal an error to SW and not start.